Update the returned Embed for the `perms` command

CalebABG opened the pull request:
Issue:
- If NOT using the `<prefix> help perms` command to get the description of the `perms` command, it could be confusing as to whether the permissions listed are for the bot itself or the user issuing the command

Resolution:
- The title of the command lists the bot's name (Silk) to let anyone requesting know the command is specifically about the bot
- There is a description below the title which is pulled from the DescriptionAttribute Attribute decorated on the method, which gives the full details as to what the command does and who it's aimed at

CalebABG committed:
Update the returned Embed for the `perms` command

Issue:
- If NOT using the `<prefix> help perms` command to get the description of the `perms` command, it could be confusing as to whether the permissions listed are for the bot itself or the user issuing the command

Resolution:
- The title of the command lists the bot's name (Silk) to let anyone requesting know the command is specifically about the bot
- There is a description below the title which is pulled from the DescriptionAttribute Attribute decorated on the method, which gives the full details as to what the command does and who it's aimed at

CalebABG commented:
Updated look w/ my guilds prefix: `silk`
![image](https://user-images.githubusercontent.com/13528490/106362520-50f7f580-62f1-11eb-9d59-94343b3c0b6f.png)

VelvetToroyashi reviewed (commented):
It might be worthwhile to use CommandsNext instead of relying on manually doing the reflection as CNext will return the description for you

CalebABG committed:
More reliable and efficient way of getting the description for the executing command

- Thank you, Velvet (forgot you could access the current Command from CommandContext)
